app/pb_kits/playbook/pb_currency/currency.rb

```diff
@@ -2,99 +2,53 @@
-                     data: default_configuration,
-                     id: default_configuration,
-                     label: default_configuration,
-                     value: default_configuration,
-                     unit: default_configuration,
-                     dollar_sign: default_configuration,
-                     size: default_configuration)
-        self.configured_align = align
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_label = label
-        self.configured_value = value
-        self.configured_unit = unit
-        self.configured_dollar_sign = dollar_sign
-        self.configured_size = size
-      end
-
-      def align
-        align_options = %w[left center right]
-        one_of_value(configured_align, align_options, "left")
-      end
-
-      def dollar_sign
-        pb_dollar_sign = Playbook::PbBody::Body.new(classname: "dollar_sign", color: "light") do
-          default_value(configured_dollar_sign, "")
-        end
-        ApplicationController.renderer.render(partial: pb_dollar_sign, as: :object)
-      end
+      partial "pb_currency/currency"
 
-      def label
-        if is_set? configured_label
-          pb_label = Playbook::PbCaption::Caption.new(text: configured_label)
-          ApplicationController.renderer.render(partial: pb_label, as: :object)
-        end
-      end
+      prop :align, type: Playbook::Props::Enum,
+                   values: %w[left center right],
+                   default: "left"
+      prop :amount, required: true
+      prop :label
+      prop :size, type: Playbook::Props::Enum,
+                  values: %w[lg sm],
+                  default: "sm"
+      prop :unit, default: "$"
 
-      def size
-        size_options = %w[lg sm]
-        one_of_value(configured_size, size_options, "sm")
+      def classname
+        generate_classname("pb_currency_kit", align_class)
       end
 
-      def value_size
+      def title_size
         size == "lg" ? 1 : 2
       end
 
-      def value
-        if is_set? configured_value
-          pb_title = Playbook::PbTitle::Title.new(size: value_size, text: configured_value, classname: "pb_currency_value")
-          ApplicationController.renderer.render(partial: pb_title, as: :object)
-        end
+      def display_amount
+        formatted_amount[1...-2]
       end
 
-      def unit
-        pb_unit = Playbook::PbBody::Body.new(classname: "unit", color: "light") do
-          default_value(configured_unit, "")
-        end
-        ApplicationController.renderer.render(partial: pb_unit, as: :object)
+      def cents
+        formatted_amount.last(2)
       end
 
-      def kit_class
-        kit_options = [
-          "pb_currency_kit",
-          align,
-        ]
-        kit_options.join("_")
-      end
+    private
 
-      def to_partial_path
-        "pb_currency/currency"
+      def align_class
```

Comment:
Since `align` can already only be these 3 strings, you are good to drop this method and just use `align` in `classname` directly.

Reply:
Awesome. Get to delete more code. Woohoo. 🎉